documentation/asciidoc/stories/assembly_connecting_remotely.adoc

```diff
@@ -0,0 +1,31 @@
+[id='remote_connections']
+:context: client
+= Connecting to {brandname} Clusters
+Connect to {brandname} via the REST or Hot Rod endpoints. You can then
+remotely create and modify cache definitions and store data across {brandname}
+clusters.
+
+.Prerequisites
+
+The examples in this section use ${SERVICE_HOSTNAME} to denote the cluster IP
+address or route hostname that provides access to your {brandname} cluster.
+
+|===
+|${SERVICE_HOSTNAME} |Client location
+
+|`.spec.clusterIP`
+|Inside {k8s}.
+
+|`.spec.clusterIP`
```

Comment:
Where did you get this from? From the simple tutorial?

For the external service it'd depend on the type of external service and even the provider. E.g. for a load balancer on AWS, that's `.status.loadBalancer.ingress\[0\].hostname`, whereas in Google/Azure I believe that'd be  `.status.loadBalancer.ingress\[0\].ip`. In NodePort it'd be actual nodes' host/port (in minikube, `minikube ip`...etc, not tried in other envs). `.spec.clusterIP` is really only relevant if you access it from inside Kubernetes, but in that case you can just use the service name itself, along with the port.

Reply:
thanks @galderz this is a nice catch. yeah I think I was getting a bit mixed up and probably started working on this section, didn't quite finish, but didn't catch it before I sent the PR. updated now.

Reply:
@galderz I would like to publish docs for 1.1.1 (already on hub). Can you approve if all is ok?

Reply:
@rigazilla I asked @galderz for review on these changes and he confirmed they look good. I'll resolve this conversation now. should be good to merge.